File: official/docs/php/orders/CreateOrder.php
<?php

\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");

$order = \EasyPost\Order::create([
    "to_address" => ["id" => "adr_..."],
    "from_address" => ["id" => "adr_..."],
    "shipments" => [
        [
            "parcel" => [
                "weight" => 10.2
            ]
        ],
        [
            "parcel" => [
                "predefined_package" => "FedExBox",
                "weight" => 17.5
            ]
        ],
    ],
]);

echo $order;

File: official/docs/go/orders/create_order.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	toAddress, _ := client.GetAddress("adr_...")
	fromAddress, _ := client.GetAddress("adr_...")
	shipment, _ := client.GetShipment("shp_...")

	order, _ := client.CreateOrder(
		&easypost.Order{
			ToAddress:   toAddress,
			FromAddress: fromAddress,
			Shipments: []*easypost.Shipment{
				shipment,
			},
		},
	)

	fmt.Println(order)
}
